Gemfile

```diff
@@ -59,6 +59,7 @@ group :development, :test do
   # We use database cleaner to empty out the database between tests (see spec_helper for usage)
   gem 'database_cleaner'
   gem 'poltergeist', '>= 1.8.1'
+  gem 'launchy' # useful for debugging rspec/capybara integration tests -- put "save_and_open_page" in your test to debug
```

Comment:
:+1: 
Can we separate the addition of this dependency into a separate commit?  That way if one or the other needs to be reverted it won't inadvertently remove something else.